src/language-css/parser-postcss.js

```diff
@@ -208,26 +214,78 @@ function parseNestedCSS(node) {
           node.source
         );
       }
+
+      return node;
     }
-    if (node.type === "css-atrule" && typeof node.params === "string") {
+
+    if (
+      node.type === "css-atrule" &&
+      typeof node.params === "string" &&
+      node.params.trim().length > 0
+    ) {
+      if (
+        node.name.toLowerCase() === "counter-style" ||
+        node.name.toLowerCase().endsWith("keyframes") ||
```

Comment:
Why does this one need `.endsWith()`?

Reply:
It’s probably for prefixed definitions, like `-webkit-keyframes`.

Reply:
@lydell prefixes `-webkit-keyframes` and etc

Reply:
Thanks! :+1: